nbextensions/config/main.js

```diff
@@ -126,10 +115,10 @@ define([
      */
     function set_config_active (extension, state) {
         state = state === undefined ? true : state;
-        console.log('nbext', state ? ' enable:' : 'disable:' , extension.Name );
+        console.log('nbext', state ? ' enable:' : 'disable:', extension.Name);
```

Comment:
Certainly, did you have anything in mind? Something like `"nbextension {name} set {not} to load for every notebook "`?